examples/codes/dubbo/client_config.json

```diff
@@ -45,12 +45,8 @@
                 {
                   "type": "proxy",
                   "config": {
-                    "downstream_protocol": "X",
-                    "upstream_protocol": "X",
```

Comment:
"upstream_protocol" is no more needed?

Reply:
yeah, it defaults to `downstream_protocol` now.